exercises/pythagorean-triplet/pythagorean_triplet.py

```diff
@@ -1,10 +1,10 @@
-def primitive_triplets():
+def primitive_triplets(nbr):
```

Comment:
Is it possible to give a more meaningful name for this parameter?

And thanks for the PR!

Reply:
@m-a-ge I've changed it to `number_in_triplet`. Could not think of a better name to go with what the function does.

Please suggest a name if this is not acceptable.